libcudacxx/include/cuda/std/__cmath/isnan.h

```diff
@@ -21,16 +21,17 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/std/__bit/popcount.h>
 #include <cuda/std/__concepts/concept_macros.h>
 #include <cuda/std/__floating_point/fp.h>
 #include <cuda/std/__type_traits/is_constant_evaluated.h>
 #include <cuda/std/__type_traits/is_integral.h>
+#if _CCCL_HAS_FLOAT128_CUDA_FUNCTIONS()
+#  include <crt/device_fp128_functions.h>
+#endif // _CCCL_HAS_FLOAT128_CUDA_FUNCTIONS()
```

Comment:
This file should be included centrally in `libcudacxx/include/cuda/std/__floating_point/nvfp_types.h`